### PR TITLE
docs: Create models.md

### PR DESCRIPTION
**Pull request by sallyqus**
camel doc - Model part

## Description
Models 

## Motivation and Context
Write our first version of camel doc.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Put the doc with right folder structure
- [ ] Move the content on notion here
- [x] Documentation (update in the documentation)
- [ ] Example (update in the folder of example)

### CODE REVIEW

**Comment by sallyqus** [comment]
Updated the model file. The table for supported models to be added.

File: docs/models.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+While CAMEL implements some of models in-house, others are integrated through third-party providers. This hybrid approach enables CAMEL to provide a comprehensive and flexible toolkit for building with LLMs.
+
+## Supported Models 
+
+The following table lists currently supported models by CAMEL.
+
+| Model Platform | Model Name | Maximum Token | Vision Modality |
+| ----- | ----- | ----- | ----- |
+| ---- | ---- | ---- | ---- |
```
Comment (Wendong-Fan) [inline docs/models.md:24]: Hey @sallyqus , will we also update this part?

**Review by sallyqus — COMMENTED** [review]
Updated the table and text in models.md

File: docs/models.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+While CAMEL implements some of models in-house, others are integrated through third-party providers. This hybrid approach enables CAMEL to provide a comprehensive and flexible toolkit for building with LLMs.
+
+## Supported Models 
+
+The following table lists currently supported models by CAMEL.
+
+| Model Platform | Model Name | Maximum Token | Vision Modality |
+| ----- | ----- | ----- | ----- |
+| OpenAI-released model | GPT_3_5_TURBO | 16385 | N |
+|OpenAI-released model| GPT_4 | 8192 | N |
```
Comment (Wendong-Fan) [inline docs/models.md]: same for belowing anthropic
```suggestion
| OpenAI| GPT_3_5_TURBO | 16385 | N |
|OpenAI| GPT_4 | 8192 | N |
```

File: docs/key_modules/models.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+You can find the in-detail code tutorial of [how to connect to LLMs](https://www.notion.so/User-Service-Agent-Discord-auto-posting-to-be-confirmed-by-Backend-team-006ee7e719a5448faa0f5987d17273e9?pvs=21), we will walk you through the process of:
+
+1. **Selecting an Open Source LLM**: Understand the strengths and capabilities of various models available in the open-source community. 
+2. **Setting Up the Environment**: Learn how to set up your development environment to integrate these models. This includes installing necessary libraries, configuring runtime environments, and ensuring compatibility with your project requirements.
+3. **Model Integration**: Step-by-step guidance on integrating your chosen LLM into your development workflow. We will cover API configurations, and leveraging pre-trained weights to maximize model performance for tasks.
```
Comment (Wendong-Fan) [inline docs/key_modules/models.md:57]: what's the additional content for this part? the link seems also wrong

**Review by Wendong-Fan — COMMENTED** [review]
Thanks @sallyqus , I made some change based on your work here: https://github.com/camel-ai/camel/pull/670/commits/99a605f01981da6f185c9be05715fb6c4951fd6e
Also for code tutorial of [how to connect to LLMs], do we have any plan to have this done?

File: docs/key_modules/models.md (hunk 3 above)
Comment (sallyqus) [reply docs/key_modules/models.md:57]: I dont think we have started this part. Who wrote this part of code? 
I have a question, is the OLLAMA integration here a part of the "how to connect to LLMs"?

**Pull request closed without merging**